### PR TITLE
Read the job parameters from env during job create

### PR DESCRIPTION
**Pull request by prachidamle**
https://github.com/rancher/cis-operator/issues/81

### CODE REVIEW

File: pkg/securityscan/job/job.go [hunk 1]
```diff
@@ -51,6 +47,9 @@ func readFromEnv(key string, defaultValue int32) int32 {
 }
 
 func New(clusterscan *cisoperatorapiv1.ClusterScan, clusterscanprofile *cisoperatorapiv1.ClusterScanProfile, clusterscanbenchmark *cisoperatorapiv1.ClusterScanBenchmark, controllerName string, imageConfig *cisoperatorapiv1.ScanImageConfig, configmapsClient wcorev1.ConfigMapController) *batchv1.Job {
+	var backoffLimit = readFromEnv("CIS_JOB_BACKOFF_LIMIT", defaultBackoffLimit)
+	var ttlSecondsAfterFinished = readFromEnv("CIS_JOB_TTL_SECONDS_AFTER_FINISH", defaultTTLSecondsAfterFinished)
```
Comment (aiyengar2) [inline pkg/securityscan/job/job.go:51]: Would it make sense to move the env var names as constants at the top of the file?

**Review by aiyengar2 — APPROVED** [review]
LGTM, nit comment

**Comment by prachidamle** [comment]
@dramich 

The reason for this was change is this conversation https://github.com/rancher/cis-operator/pull/77#discussion_r542975308
However, I tested this and even if we now  read the env during job create - to change the env or set new values, I still need to restart the operator.

so not sure if what I am doing is achieving that or is necessary

**Comment by prachidamle** [comment]
@dramich responded offline and agree that this change is not avoiding the cis-operator pod restart. So we need not do this.

**Pull request closed without merging**